### ISSUE TITLE
How to handle duplicate names in codegen output?

### ISSUE THREAD

**Issue by orva** [diwic/dbus-rs#235]
On my system with NetworkManager 1.22.6 codegen produces invalid code with following incantation:

```
dbus-codegen-rust --system-bus --generic-variant --methodtype None  \
    --destination org.freedesktop.NetworkManager                                  \
    --path /org/freedesktop/NetworkManager
```

Output is invalid because following methods are defined twice:
```
    fn get_devices(&self) -> Result<Vec<dbus::Path<'static>>, dbus::Error>;
    fn get_all_devices(&self) -> Result<Vec<dbus::Path<'static>>, dbus::Error>;
```

Reason is pretty simple: dbus interface contains methods `GetDevices` and `GetAllDevices` + properties `Devices` and `AllDevices`.

Obviously I can just delete/rename duplicate methods from the generated output, but I wonder if there is/should be way to instruct codegen not to generate those duplicate names?

**Comment by diwic** [comment]
Hi!

I believe this was resolved some time back, but I never pushed a release. I have now released v0.5.0 of dbus-codegen, could you try it and if it does not resolve the issue, reopen this bug? Thanks!

**Comment by orva** [comment]
It does indeed fix the problem. To be honest, this was how I really wanted it to be fixed, even though it was backwards incompatible. Thank you!
